values-schema.yaml

```diff
@@ -1292,6 +1292,17 @@ properties:
           enabled:
             type: boolean
             default: true
+      psp-allowed-repos:
+        additionalProperties: false
+        required: [enabled]
+        properties:
+          repos:
+            type: array
+            items:
+              type: string
+          enabled:
+            type: boolean
+            default: true
```

Comment:
default should be false...c'mon, don't be so sloppy

Reply:
no! that is not what is intended here.
If you specify the policy, you must force the `enabled` field to become `true`.
Not specifying the policy or the enabled field for the policy is actually the same as disabling the policy.

Reply:
when using these values with autocomplete in VScode, it is more user friendly to have the `enabled: true` value in each policy pre-filled for you.

Reply:
ok, good point